composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ybelenko/openapi-data-mocker-server-middleware",
+    "name": "anttikuuskoski/openapi-data-mocker-server-middleware",
```

Comment:
Mmm, that's not how opensource development works. Imagine if every contributor will swap original repo with his  own fork, it doesn't make sense because next contributor need to fork from your fork etc.

I just need to merge your changes and issue new patch release version(`1.0.1`) so packagist will be synced. To update package in your project you can use command from [Composer - Updating dependencies to their latest versions](https://getcomposer.org/doc/01-basic-usage.md#updating-dependencies-to-their-latest-versions) or you can write it manually in your `composer.json` as `"ybelenko/openapi-data-mocker-server-middleware": "^1.0.1"`.

I agree with your changes regarding `false/null` warning and will fix it soon right in this PR. Thanks for the great effort.

Reply:
My apologies, I never meant for the latter change (committed for my personal needs) to end up in the pull request. I did not gather that commits after the PR would be included in the PR (duh). 

I forcefully reverted the commit in any case so the PR should be usable. Or would you prefer a new PR?

Thank you for the learning and friendly attitude!